src/_variables.scss

```diff
@@ -16,6 +16,19 @@ $wb-assets-path: "#{$wb-core-path}/assets" !default;
 // BOOTSTRAP VARIABLES
 //
 
+//== Breadcrumb
+//
+//## Breacrumb enable/disable and colour settings.
+
+// WET Custom
+$wb-bc-enable: true !default; // Enable or disable WET's breadcrumb SCSS. Themes that don't use breadcrumbs can set this to false.
+$wb-bc-background-color: null !default; // Outer background colour. Override $breadcrumb-bg (Bootstrap variable) to change the inner background colour.
+$wb-bc-separator-color: #333 !default; // Separator colour.
+$wb-bc-separator-font-family: "Glyphicons Halflings" !default; // Separator font family.
+$wb-bc-separator-font-size: 0.7em !default; // Separator font size.
+$wb-bc-separator-content: "\e092" !default; // Separator content.
+$wb-bc-separator-content-rtl: "\e091" !default; // Separator content (right to left).
+
```

Comment:
That block is for general overrides wjile the section where you moved it is for component specific variables. In the output it makes no difference but I wanted that file to be well organized, even if we don't have any formal specification for it.